### ISSUE TITLE
Svelteapp not building on main

### ISSUE THREAD

**Issue by DamnClin** [jhipster/jhipster-lite#3839]
Svelteapp not building on main because of bad coverage : 

```
[INFO] PASS src/test/javascript/spec/common/primary/app/App.spec.ts
[INFO]   ✓ Welcome the user (15 ms)
[INFO] 
[INFO] 
[INFO] =============================== Coverage summary ===============================
[INFO] Statements   : 96.22% ( 102/106 )
[INFO] Branches     : 33.33% ( 4/12 )
[INFO] Functions    : 70% ( 7/10 )
[INFO] Lines        : 98% ( 98/100 )
[INFO] ================================================================================
[INFO] Jest: "global" coverage threshold for statements (100%) not met: 96.22%
[INFO] Jest: "global" coverage threshold for branches (100%) not met: 33.33%
[INFO] Jest: "global" coverage threshold for lines (100%) not met: 98%
[INFO] Jest: "global" coverage threshold for functions (100%) not met: 70%
[INFO] Test Suites: 1 passed, 1 total
[INFO] Tests:       1 passed, 1 total
[INFO] Snapshots:   0 total
[INFO] Time:        1.667 s, estimated 2 s
[INFO] Ran all test suites.
```

**Comment by qmonmert** [comment]
Fix with vitest #3873 
<img width="594" alt="image" src="https://user-images.githubusercontent.com/9989211/194770673-ebc9fa30-775e-451f-918d-2e24682023f0.png">

**Comment by pascalgrimaud** [comment]
Nice @qmonmert : so moving to vitest is fine
